api/models/risk_of_bias.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const bookshelf = require('../../config').bookshelf;
+const BaseModel = require('./base');
+require('./source');
+require('./trial');
+require('./risk_of_bias_criteria')
+require('../helpers');
+
+const relatedModels = [
+  'source',
+  'trial',
+  'risk_of_bias_criteria'
+];
+
+const RiskOfBias = BaseModel.extend({
+  tableName: 'risk_of_biases',
+  hasTimestamps: true,
+  visible: [
+    'id',
+    'trial_id',
+    'source_id',
+    'study_id',
+    'source_url',
+    'created_at',
+    'updated_at'
+  ],
+  source: function () {
+    return this.belongsTo('Source');
+  },
+  trial: function() {
+    return this.belongsTo('Trial');
+  },
```

Comment:
We don't use the `source` or `trial` relationships anywhere yet, so please remove those.